packages/airnode-starter/integrations/coingecko/create-secrets.ts

```diff
@@ -0,0 +1,23 @@
+import { writeFileSync } from 'fs';
```

Comment:
I understand that we kinda need different `create-secrets` and `make-request` for each integration but we also kinda don't :smile: Most of the code will be pretty much the same, won't it? Maybe providing some helper functions instead just using the same code would be nice.

Reply:
Same goes for `Requester` contract.

Reply:
I'll think about this.

Reply:
I have simplified this. There is an utility file common for all integrations now.